pybamm/solvers/processed_variable_var.py

```diff
@@ -0,0 +1,437 @@
+#
+# Processed Variable class
+#
+import casadi
+import numpy as np
+import pybamm
+from scipy.integrate import cumulative_trapezoid
+import xarray as xr
+
+
+class ProcessedVariableVar(object):
```

Comment:
I think we need to explain the differences to `ProcessedVariable` in the docstring. Also, I'm unsure what the last `Var` in `ProcessedVariableVar` means? `Variant`?

Reply:
The class has now been renamed to `ProcessedVariableComputed` and the docstring updated to specify the purpose of the variant class.